### ISSUE TITLE
Finetune datasets and parameters based on sft-trained model

### ISSUE THREAD

**Issue by maxfanxd** [AnjieCheng/NaVILA#46]
I tried the data mixture: r2r+rxr+scanqa+human(all of these I sampled 2% for them) to finetune the model navila-llama3-8b-8f (already sft-trained) with low learning rate about 5e-5, getting worse success rate while evaluating. Then I tried to delete the r2r dataset getting similar results.
Since I want to construct my own dataset to finetune this model for special scenes, I wonder how to mix my dataset and which special samples I should augment. I'd strongly appreciate it if you could give some advice on learning rate or other parameters!

**Comment by Monda-Hou** [comment]
Hello! I have noticed that you have achieved the evaluation process during your work. I'm struggling on the evaluation. Could you please give me some tips on it? I have questions as below, I'll appreciate it if you could give some tips
1. Which datasets? r2r, rxr, envdrop? I guess that the former two will work, but not sure about it.
2. How to make evaluation? Just follow the EVALUATION process on the NaVILA program website, or there still need the config files or codes?
3. Use the pretrain model or the model has been trained?
